frontend/demo/src/pages/Tags/Tag.module.scss

```diff
@@ -0,0 +1,74 @@
+.tag {
+  display: inline-block;
+  padding: 4px;
+  max-width: 100%;
+}
+
+.tagInner {
+  display: inline-flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0px 8px;
+  background: var(--color-airy-blue);
+  color: #fff;
+  border-radius: 16px;
+  line-height: 24px;
+  max-width: 100%;
+  white-space: nowrap;
+
+  &:hover {
+    word-break: break-all;
+    white-space: normal;
+    line-height: 16px;
+    padding: 4px 8px;
+  }
+
+  &.clickable {
+    cursor: pointer;
+  }
+}
+
+.removeTag {
+  cursor: pointer;
+  margin-bottom: auto;
+}
+
+.closeButton {
+  display: inline-block;
+  padding-left: 4px;
+
+  svg {
+    path {
+      fill: #fff;
+    }
+  }
+}
+
+.tagName {
+  min-width: 0;
+  flex: 1;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+.tagNameExpanded {
+  @extend .tagName;
+  white-space: unset;
+  overflow-wrap: break-word;
+  margin-right: 4px;
+}
+
+.#{tag-} {
```

Comment:
Could you explain this selector to me? 🤨  I've genuinely never seen this and would like to learn!

Reply:
@chrismatix @bitboxer wrote this code and tbh I just took it as it was, cause it looked well organized!